### PR TITLE
Update example_centos.rst for PHP8.2

### LINKED ISSUE #9652: Update centos installation example
It’s using PHP 7.4 which is not supported anymore
Ref: https://github.com/nextcloud/documentation/pull/9650#issuecomment-1438635305

### PR DESCRIPTION
**Pull request by hellodarkness**
### ☑️ Resolves

* Fix #9652 

### 🖼️ Screenshots

centos

### CODE REVIEW

File: admin_manual/installation/example_centos.rst [hunk 1]
```diff
@@ -79,24 +79,29 @@ Command to install the yum-utils package (for the yum-config-manager command):
...
 
 
-
 Installing PHP and the required modules
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Next install the PHP modules needed for this install. Remember, because this is a limited basic install, we only install the neccessary modules, not all of them. If you are making a more complete install, please refer to PHP module list in the source installation documentation, :doc:`../installation/source_installation`.::
+Next, install the PHP modules needed for this install. Remember, because this is a limited basic install, we only install the neccessary modules, not all of them. If you are making a more complete install, please refer to PHP module list in the source installation documentation, :doc:`../installation/source_installation`::
 
     dnf install -y php php-gd php-mbstring php-intl php-pecl-apcu\
```
Comment (kesselb) [inline admin_manual/installation/example_centos.rst:96]:
```suggestion
    dnf install -y php php-cli php-gd php-mbstring php-intl php-pecl-apcu\
```

File: admin_manual/installation/example_centos.rst [hunk 2]
```diff
@@ -79,24 +79,29 @@ Command to install the yum-utils package (for the yum-config-manager command):
...
          php-mysqlnd php-opcache php-json php-zip
 
+It may also be useful to install the php-cli module, to ease future troubleshooting:
+
+::
+
+    dnf install php-cli
```
Comment (kesselb) [inline admin_manual/installation/example_centos.rst:103]:
```suggestion
```

**Review by kesselb — CHANGES_REQUESTED** [review]
Thanks :+1:

**Comment by hellodarkness** [comment]
> Thanks 👍

I'd originally kept php-cli separate as it's optional but fair point, your way is more concise!

**Comment by kesselb** [comment]
> php-cli separate as it's optional

You cannot use occ without, and therefore it's not really optional ;)

**Review by kesselb — APPROVED** [review]

**Pull request merged**